### PR TITLE
Create ability to write tasks from CheatLogs

### LINKED ISSUE #15: First Iteration of writing data from the CheatLogs Application

### PR DESCRIPTION
**Pull request by theopin**

### CODE REVIEW

File: src/main/java/storage/WriteDataFile.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static storage.ParseCheatSheetFile.NAME;
+import static storage.ParseCheatSheetFile.PROGRAMMING_LANGUAGE;
+import static storage.ParseCheatSheetFile.DETAILS;
+
+/**
```
Comment (abnermtj) [inline src/main/java/storage/WriteDataFile.java:16]: perhaps remove  some of the old Duke descriptions

**Review by brandontoh — APPROVED** [review]
Awesome work!

File: src/main/java/storage/WriteDataFile.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        executeFunction();
+    }
+
+    /**
+     * Converts the tasks present in the list of tasks into
+     * a string. This string is then saved in a form of .txt file
+     * into documentTracker.txt.
+     */
+    @Override
+    public void executeFunction() {
```
Comment (abnermtj) [inline src/main/java/storage/WriteDataFile.java:34]: perhaps a more apt name than executeFunction(), idk

File: src/main/java/storage/WriteDataFile.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        */
+    }
+
+    /**
+     * Converts the cheatSheet into a file format.
+     *
+     * @param cheatSheet The cheatSheet that is currently being converted into a file.
+     */
+    private void convertStringToFile(CheatSheet cheatSheet) {
+
```
Comment (abnermtj) [inline src/main/java/storage/WriteDataFile.java]: maybe remove the spacing between first line of the method and the next, for consistency

**Review by abnermtj — CHANGES_REQUESTED** [review]

File: src/main/java/storage/WriteDataFile.java (hunk 1 above)
Comment (brandontoh) [reply src/main/java/storage/WriteDataFile.java:16]: hmm isn't that his own functions and not the old Duke functions?

Comment (abnermtj) [reply src/main/java/storage/WriteDataFile.java:16]: er like the line i commented on said duke.txt and list of tasks

**Review by abnermtj — APPROVED** [review]

**Pull request merged**